backend_fen.go

```diff
@@ -266,8 +269,15 @@ func (w *Watcher) AddWith(name string, opts ...addOpt) error {
 
 // Remove stops monitoring the path for changes.
 //
-// Directories are always removed non-recursively. For example, if you added
-// /tmp/dir and /tmp/dir/subdir then you will need to remove both.
+// If the path was added as a recursive watch (e.g. as "/tmp/dir/...") then the
+// entire recusrice watch will be removed. You can use both "/tmp/dir" and
+// "/tmp/dir/..." (they behave identical).
```

Comment:
```suggestion
// If the path was added as a recursive watch (e.g. as "/tmp/dir/...") then the
// entire recursive watch will be removed. You can use either "/tmp/dir" or
// "/tmp/dir/..." (they behave identically).
```

Reply:
I see you left comments on all of these; that's not really necessary: the documentation for the `Watcher` types are all generated from mkdoc.zsh (it's not obvious at all its generated, maybe I should add a comment at the top of the file). I'll update it from there. (Also: I just have been asleep when I wrote this...)